src/Farmer/Arm/Web.fs

```diff
@@ -392,6 +392,17 @@ type Certificate =
                            canonicalName = this.DomainName |}
                 |} :> _
 
+type SlotConfigName = 
+    { SiteName : ResourceName
+      SlotSettingNames: List<string> }
```

Comment:
Should this be a `string Set` instead? It seems you shouldn't be allowed to add the same name more than once.

Reply:
Update it to be string set. The order of items wasn't consistent, on the tests started using containsAll instead of sequenceEqual